### DISCUSSION TITLE
Question about Computing Resources

### DISCUSSION THREAD

**Discussion by cheshmi** [OpenCilk/opencilk-project#200] [General]
Hello,

I am exploring various cloud computing options for the course. I noticed that you used AWS for the course, and I am curious about how you handled it. How did you pay for AWS? Does Amazon have any educational plan (different than $100 credit)? How many resources did you allocate for each student?

Thanks,

**Comment by jowens** [comment]
At UC Davis, we (the university) have a campus-wide contract with Amazon (perhaps this is a contract over all 10 UC campuses, not sure). We used this contract, which I presume and believe has competitive pricing. The benefit of getting under this contract's umbrella is that UC has streamlined billing with Amazon and so I didn't have to (as an instructor) worry about getting a bill and paying it, or having students be cognizant/responsible for their own billing. We were able to set up each student with a path to submit jobs to AWS so that all the billing was handled automatically and went to a common account associated with our department.

We spent $155 per enrolled student on compute time, which was in line with my (and MIT instructors') expectations.

**Comment by cheshmi** [comment]
Thanks John.
